### ISSUE TITLE
Feature Request: Select all messages in the bundle

### ISSUE THREAD

**Issue by vtexperts** [teresa-ou/inboxy#3]
I had a bundle with 20 messages and only 2 of them I wanted to keep, I had to check 18 messages manually and archive them.

It would be nice to have checkbox option to "Select All" in the bundle, so the user can uncheck the ones to keep and process the rest.

Thanks!

**Comment by teresa-ou** [comment]
Thanks, I can look into this! 

In the meantime, another way to quickly select multiple messages is to use the Shift key + click.

**Comment by vtexperts** [comment]
Thanks!

I tried shit + left click, but it does nothing (if clicking on checkbox) and opens new window for that email if clicking on the row.

Is it left click?

**Comment by teresa-ou** [comment]
Yes, you would left click the checkbox for an email, and then shift + left click the checkbox for another email. All emails between those two (inclusive) will be selected.

The first part of this article shows the steps with screenshots: https://www.lifewire.com/how-to-select-multiple-messages-quickly-in-gmail-1172037

**Comment by vtexperts** [comment]
I can't believe I did not know this! Thanks!

**Comment by benoror** [comment]
Hi! Great request. Upstream inboxy is largely unmaintained, so this lives on in **Inbundly** (my GPL-3.0 fork of Teresa Ou's inboxy).

**Select all in a bundle** shipped in Inbundly v4.2.0: each bundle row has a checkbox that selects or deselects every message in the bundle (Gmail-style, indeterminate when partial). Then use Gmail's toolbar for Archive, Delete, Mark as read, Move, etc. **Archive-all** is also on by default.

If you specifically want a one-click **Delete all** button on the row (not just select then Delete), please open that as a feature request. Happy to prioritize with a clear write-up.

Chrome: https://chromewebstore.google.com/detail/inbundly/pbfjicjmcpogjlbpljebhhgkgfhbdcga
Firefox: https://addons.mozilla.org/firefox/addon/inbundly/
Site: https://inbundly.com/
Issues: https://github.com/benoror/inbundly/issues
